entities/entities/fadmin_motd/cl_init.lua

```diff
@@ -12,7 +12,7 @@ local function WorldToScreen(vWorldPos,vPos,vScale,aRot)
 end
 
 function ENT:LoadPage()
-	local Page = GetConVarString("_FAdmin_MOTDPage")
+	local Page = GetConVar("_FAdmin_MOTDPage"):GetString()
```

Comment:
I wouldn't be surprised if the definition of GetConvarString is literally the C variant of  `return GetConvar(x):GetString()`